### DISCUSSION TITLE
OH MY POSH not working in vscode terminal

### DISCUSSION THREAD

**Discussion by Uyaii** [JanDeDobbeleer/oh-my-posh#5722] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I reccently changed the path of my OMP themes so when i opened my windows terminal and vscode terminal the theme was different but now i've made the changes in the profile document and it works in my normal windows terminal but it doesnt work in my vscode terminal, if i invoke it directly there it works but when i open another instance of the terminal it reverts back to the old theme
![rightterminal](https://github.com/user-attachments/assets/88a6d345-a751-4887-8e6b-55938ab93ed6)
![wrongterminal](https://github.com/user-attachments/assets/0a1e32a1-3725-41e9-b3ae-ea94e144c85c)

### Theme

im using the dracula theme

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 19.11.4

Shell: pwsh (7.4.5)

Prompt:


Segments:

ConsoleTitle(true) -   1 ms
Session(true)      -   0 ms
Path(true)         -   1 ms
Git(true)          - 183 ms
Root(false)        -   0 ms
Status(true)       -   1 ms
Node(false)        -   2 ms
Go(false)          -   1 ms
Python(false)      -   2 ms
Shell(true)        -   0 ms
Time(true)         -   0 ms

Run duration: 208.1562ms

Cache path: C:\Users\HP PC\AppData\Local\oh-my-posh

Config path: no --config set, using default built-in configuration
```

**Comment by JanDeDobbeleer** [comment]
@Uyaii this is going to be a configuration issue. What does the profile look like in both instances? They’re not necessarily the same.

**Reply by Uyaii** [reply]
i dont know how to open the profile file from the vscode terminal but this is how it looks like when i open it from my windows terminal
![profile](https://github.com/user-attachments/assets/042965d6-3cae-47b6-8819-c7f8b13508f6)

**Reply by JanDeDobbeleer** [reply]
@Uyaii You can always do `Get-Content $PROFILE` from the PowerShell instance to get the contents.

**Reply by Uyaii** [reply]
okay ive seen the problem but i dont know why it's that way especially when in the original file the second line isnt commented out. it actually had a totally different configuration, thank you
